packages/datagateway-download/src/downloadStatus/downloadProgressIndicator.component.tsx

```diff
@@ -0,0 +1,66 @@
+import { Box, LinearProgress, Typography } from '@mui/material';
+import React from 'react';
+import {
+  useDownload,
+  useDownloadPercentageComplete,
+} from '../downloadApiHooks';
+import { useTranslation } from 'react-i18next';
+
+interface DownloadProgressIndicatorProps {
+  downloadId?: number;
+}
+
+/**
+ * A component for showing the download progress of the given download item.
+ * @param download The download item that this component should show the progress of.
```

Comment:
This comment doesn't accurately portray that it takes a `downloadId`, not a `download`